components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/hardwareresourcechecktrigger/hardwareresourcecountchecktrigger.go

```diff
@@ -28,7 +28,7 @@ func NewHardwareResourceCountCheck(log logger.Logger, port string) *HardwareReso
 	}
 }
 
-func (ss *HardwareResourceCountCheck) Run(config models.Config) []models.CheckTriggerResponse {
+func (ss *HardwareResourceCountCheck) Run(config *models.Config) []models.CheckTriggerResponse {
 	ss.log.Info("Performing Hardware Resource count check from batch check ")
```

Comment:
Same here, there should be some validations? aren't we expecting that this won't be skipped at all?